prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -317,27 +317,68 @@ int64_t rxr_pkt_req_cq_data(struct rxr_pkt_entry *pkt_entry)
 	return cq_data_hdr->cq_data;
 }
 
-size_t rxr_pkt_req_max_header_size(int pkt_type)
+
+size_t rxr_pkt_calc_data_size(struct rxr_ep *ep,
```

Comment:
The function name should be `rxr_pkt_req_data_size` (add `req`) as it only handle req packet types